tests/pytests/test_config.py

```diff
@@ -859,6 +859,74 @@ def _testImmutableStringConfig(env, configName, ftConfigName, ftDefault,
         _testImmutableStringConfig(env, configName, ftConfigName, ftDefault,
                                    testValue)
 
+
+@skip(cluster=False, redis_less_than='8.1.0')
+def testConfigAPIRunTimeOssGlobalPassword():
+    env = Env(noDefaultModuleArgs=True)
+    if env.env != 'oss-cluster':
+        env.skip()
+
+    env.expect('CONFIG', 'GET', 'search-oss-global-password')\
+        .equal(['search-oss-global-password', 'Password: *******'])
+
+    env.expect('CONFIG', 'SET', 'search-oss-global-password', '123')\
+        .error().contains('CONFIG SET failed')
+
+
+@skip(cluster=False, redis_less_than='8.1.0')
```

Comment:
Should be greater than 8.0.2? (what we have now)

Reply:
Should be skipped on less than `8.0`, for redis-server 8.0.m02 the version number is `v=7.9.225`